### DISCUSSION TITLE
Z-stop wiring

### DISCUSSION THREAD

**Discussion by Ixian** [dustin-range/3D_Printer#2] [General]
Curious how you run the wires for the Z-stop - do you rig them on the plate over to the Y cable chain or....?

**Comment by dustin-range** [comment]
I ran them between the bed and the carriage. Then into the Y cable chain. They basically lay there. I tried to zip tie them where possible to the carriage.

It's not perfect, but I have not had them tangle or give me any problems.

**Reply by Ixian** [reply]
Thanks! I'll give that a go.
